### PR TITLE
feat/created APIs for contract

### PR DESCRIPTION
**Pull request by shravan-n**
Created APIs for contract table with Exception handling

### CODE REVIEW

File: backend/src/main/java/com/seeder/contract/controller/ContractController.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+@RequestMapping("/contracts")
+public class ContractController {
+
+    @Autowired
+    private ContractService contractService;
+
+    @Autowired
+    private ContractMapper contractMapper;
+
+    public void checkContractId(int contractId) throws ContractNotFoundException{
```
Comment (Thota-Aakanksha) [inline backend/src/main/java/com/seeder/contract/controller/ContractController.java:26]: helper methods can be private

File: backend/src/main/java/com/seeder/contract/controller/ContractController.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        return contractMapper.convertToContractDTO(contract.get());
+    }
+
+    @PostMapping("")
+    public ContractDTO saveContract(@RequestBody ContractDTO contractDTO){
+        Contract contract=contractMapper.convertToContract(contractDTO);
+        return contractMapper.convertToContractDTO(contractService.saveContract(contract));
+    }
+
+    @PutMapping("")
```
Comment (Thota-Aakanksha) [inline backend/src/main/java/com/seeder/contract/controller/ContractController.java:56]: put requires an id right ?

**Review by Thota-Aakanksha — CHANGES_REQUESTED** [review]

File: backend/src/main/java/com/seeder/contract/controller/ContractController.java (hunk 1 above)
Comment (shravan-n) [reply backend/src/main/java/com/seeder/contract/controller/ContractController.java]: changed to private

File: backend/src/main/java/com/seeder/contract/controller/ContractController.java (hunk 2 above)
Comment (shravan-n) [reply backend/src/main/java/com/seeder/contract/controller/ContractController.java]: yes.. changed

**Review by Thota-Aakanksha — APPROVED** [review]

**Pull request merged**